src/DependencyInjection/Factory/BlockAdminFactory.php

```diff
@@ -70,9 +73,11 @@ public function create(array $config, ContainerBuilder $container, XmlFileLoader
 
         $bundles = $container->getParameter('kernel.bundles');
 
-        if (true === $config['enable_menu']
-            || ('auto' === $config['enable_menu'] && isset($bundles['CmfMenuBundle']))
-        ) {
+        $message = 'CmfMenuBundle integration was explicitely enabled, but the bundle is not available';
+        if (class_exists(CmfMenuBundle::class)) {
+            $message .= ' (did you forget to register the bundle in the AppKernel?)';
+        }
+        if ($this->isConfigEnabledAuto($container, $config['enable_menu'], 'CmfMenuBundle', $message.'.')) {
```

Comment:
absolutely. i just find the way we create the exception message a bit weird, with adding the dot at the end instead of just adding the hint as second sentence. really just a detail.

we seem to have an xml schema mismatch, if i read the tests correctly. can you adress that please?